include/psa/crypto_values.h

```diff
@@ -608,7 +608,17 @@
  */
 #define PSA_ALG_VENDOR_FLAG                     ((psa_algorithm_t)0x80000000)
 
-#define PSA_ALG_CATEGORY_MASK                   ((psa_algorithm_t)0x7f000000)
+/** Policy-only flag.
```

Comment:
Constants such as `xxx_FLAG` are only meant as helper in the implementation and are not considered part of the PSA Crypto API specification nor of the public API of Mbed TLS. They must not be documented in Doxygen. A comment starting with `/*` is fine.

Reply:
N/A now that the policy bit is removed.